packages/backend/src/controllers/house/dto/create-task.dto.ts

```diff
@@ -0,0 +1,22 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { TaskModel } from '../../../db/task/task.schema';
+
+export class CreateTaskDto extends TaskModel {
+  @ApiProperty()
+  readonly name: string;
+
+  @ApiProperty()
+  readonly description: string;
+
+  @ApiPropertyOptional()
+  readonly last_completed: Date;
```

Comment:
probably remove this

Reply:
as in the field or change the annotation to ApiProperty

Reply:
yeah sorry, remove the field. I can't imagine a situation where you would create a task that is already complete